erpnext/shopping_cart/cart.py

```diff
@@ -87,9 +88,14 @@ def place_order(confirmed):
 	return quotation.name
 
 @frappe.whitelist()
-def update_cart_item(item_code, fieldname, value, with_items=False):
+def update_cart_item(item_code, fieldname, value, with_items=False, name=None):
 	from erpnext.stock.get_item_details import get_conversion_factor
-	quotation = _get_cart_quotation()
+
+	if name:
```

Comment:
This is unnecessary. You can pass name and if it is None then it'll be treated as having no name